### PR TITLE
own KBD67 MKII keymap

### PR DESCRIPTION
**Pull request by cykedev**
added own keymap

### CODE REVIEW

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/cykedev/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#undef DISABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/cykedev/config.h:1]:
```suggestion
#pragma once

#undef DISABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
```

**Review by drashna — CHANGES_REQUESTED** [review]
could you add a GPL2+ license header to the config.h and keymap.c files?

**Comment by cykedev** [comment]
> could you add a GPL2+ license header to the config.h and keymap.c files?

done

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
> done

I don't see any changes for it?  by chance, forgot to push the changes?

**Comment by cykedev** [comment]
> > done
> 
> I don't see any changes for it? by chance, forgot to push the changes?

Ah I see I messed it up. Should be done now

**Review by mrsendyyk — APPROVED** [review]

**Review by drashna — APPROVED** [review]
LGTM, thanks!

**Pull request merged**